### PR TITLE
ARK-42 Core Arclight Component

### PR DESCRIPTION
**Pull request by GeronimoJohn**
- Found a way to create a custom block on Wordpress through the plugin Lazyblocks - this custom block is called Arclight
- The custom block arclight renders arclight videos on Wordpress and enables us to pull data from GraphQL
- Created a component for Arclight
- Depends on #32

### CODE REVIEW

**Comment by tataihono** [comment]
@GeronimoJohn can you write `depends on #32` so that its clear what PR this is connected with?

File: src/elements/block/core-embed/embed/error/error.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import React from 'react'
```
Comment (tataihono) [inline src/elements/block/core-embed/embed/error/error.spec.tsx:1]: this file needs to go.

File: src/elements/block/core-embed/embed/error/error.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import React from 'react'
```
Comment (tataihono) [inline src/elements/block/core-embed/embed/error/error.tsx:1]: this file needs to go.

File: src/elements/block/core-embed/embed/error/index.ts [hunk 3]
```diff
@@ -0,0 +1,2 @@
+export { Error } from './error'
```
Comment (tataihono) [inline src/elements/block/core-embed/embed/error/index.ts:1]: this file needs to go.

File: src/elements/block/core-paragraph/core-paragraph.tsx [hunk 4]
```diff
@@ -14,7 +14,7 @@ export type CoreParagraphProps = {
   __typename: 'CoreParagraphBlock'
 }
 
-export function CoreParagraph({ attributes }: CoreParagraphProps) {
+export function CoreParagraph({ attributes, __typename }: CoreParagraphProps) {
```
Comment (tataihono) [inline src/elements/block/core-paragraph/core-paragraph.tsx:17]: unnecessary change

**Review by tataihono — CHANGES_REQUESTED** [review]
there are a bunch of out-of-scope changes in this PR. Please remove them thanks!

File: src/elements/block/core-arclight/core-arclight.spec.tsx [hunk 5]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import { BasicCoreArclight } from './core-arclight.composition'
+
+it('should render with the correct text', () => {
```
Comment (tataihono) [inline src/elements/block/core-arclight/core-arclight.spec.tsx:5]: this is not a good summary of what this test is doing.

**Review by tataihono — CHANGES_REQUESTED** [review]
there are a number of unrelated changes in this PR. can you please back them out.

**Review by tataihono — CHANGES_REQUESTED** [review]
still have a bunch of unrelated changes in this branch @GeronimoJohn

**Comment by GeronimoJohn** [comment]
Too many out of scope changes in this branch. Will be replaced by a new branch

**Pull request closed without merging**